backend/Origam.Workbench/Commands/SchemaCommands.cs

```diff
@@ -495,6 +549,7 @@ public EditSchemaItem(bool showDialog)
         }
 
         public bool ShowDialog { get; set; }
+        public bool ShowDocked { get; set; }
```

Comment:
Is this the right naming for the desired function? If `ShowDocked` means that a property grid editor will open, then I would create a completely separate command that just opens a property grid editor instead of the general edit command and baking in another `if` statement.